### DISCUSSION TITLE
Windows installer?

### DISCUSSION THREAD

**Discussion by Biggly** [danny-avila/LibreChat#2567] [Q&A] [answered]
All I see is codey stuff, for coders?

*blinks

**Comment by fuegovic** [comment, marked as answer]
the easiest way to get going, is to use docker
source: https://docs.librechat.ai/install/installation/docker_compose_install.html

If you don't want to go through this but want to experience LibreChat, feel free to use our demo: https://demo.librechat.cfd/

### Docker quick start:
Here's the quick summary to get started with the default configuration:
> Requirement: `Git` and `Docker`
  
- Clone the repo
```bash
  git clone https://github.com/danny-avila/LibreChat.git
```
- navigate to the LibreChat folder
```bash
  cd LibreChat
```
- Create a .env from the .env.example
  - note: you might need to use `copy` instead of `cp` if you're using Windows 10
```bash
  cp .env.example .env
```
- Start LibreChat
```sh
  docker compose up -d
```
- Access LibreChat
> visit [http://localhost:3080/](http://localhost:3080/)

**Reply by Biggly** [reply]
I've marked that as the correct answer, and thanks!

(I have no idea what you're talking about and will stick with Faraday.dev and LM Studio, but thanks anyway)
